bootstrap/spigot/build.gradle.kts

```diff
@@ -1,4 +1,4 @@
-val paperVersion = "1.17.1-R0.1-SNAPSHOT" // Needed because we do not support Java 17 yet
+val paperVersion = "1.19-R0.1-SNAPSHOT"
 val viaVersion = "4.0.0"
 val adaptersVersion = "1.4-SNAPSHOT"
```

Comment:
This was updated in 4a9581c5a62d9b84b6f8b55ddce98333e80f204f.

Reply:
https://github.com/GeyserMC/Geyser/pull/3131/commits/0335c8263c21e30a28baa0f34a9eb97dbe808de3#diff-dad23bc40c6a07b714bb844df276d9123869a23302293658d656f926d0805cf8R3